source/technical/swift.rst

```diff
@@ -203,36 +203,76 @@ ________________________________________
+  
+  .. code-block:: bash
+
+     # using keystone v2
+     username=<username>
+     password=<password>
+     tenant=<project name>
+     region=<region name> # CHI@TACC or CHI@UC
+     authurl=https://chi.<uc/tacc>.chameleoncloud.org:5000/v2.0
+   
+     # using keystone v3
+     username=<username>
+     password=<password>
+     projectid=<project id>
+     region=<region name> # CHI@TACC or CHI@UC
+     authurl=https://chi.<uc/tacc>.chameleoncloud.org:5000/v3
+
+3. Pass Chameleon credentials as command line options (see below)
+
+To mount, use the following command:
 
 .. code-block:: bash
 
-   cloudfuse <mount_dir>
+   cc-cloudfuse mount <mount_dir>
 
-Or you can specify ``username``, ``password``, ``tenant``, ``region`` or ``authurl`` as mount options:
+If you don't use :ref:`Chameleon RC file <cli-rc-script>` or ``~/.cloudfuse`` file, you can pass your Chameleon credentials as command line options:
 
 .. code-block:: bash
 
-   cloudfuse -o username=<username>,password=<password> <mount_dir>
+   # using keystone v2
+   cc-cloudfuse mount <mount_dir> -o username=<username>,password=<password>,tenant=<project name>,region=<region name>,authurl=<auth url v2.0>
+   
+   # using keystone v3
+   cc-cloudfuse mount <mount_dir> -o username=<username>,password=<password>,projectid=<project id>,region=<region name>,authurl=<auth url v3>
 
 Now you can access your Chameleon Object Store as your local file system.
 
-To unmount:
+To unmount, use the following command:
 
 .. code-block:: bash
 
-   fusermount -u <mount_dir>
+   cc-cloudfuse unmount <mount_dir>
+   
+.. Important::
+   **Limitations**
+   
+   The primary usage scenario of the ``cc-cloudfuse`` tool is to allow you to interact with Chameleon Object Store using familiar file system operations. 
+   Because the ``cc-cloudfuse`` runs on top of an object store, it is important to understand that not all functionality will behave identically to a regular file system. 
+   
+   #. Symbolic links, file permissions, and POSIX file locking operations are not supported.
+   #. Updating an existing file is an expensive operation as it downloads the entire file to local disk before it can modify the contents.
+   #. You can mount from multiple nodes, but there is no synchronization between nodes regarding writes to Object Storage.
+   #. The mounting root directory can only contain directories, as they are mapped to Object Store containers.
+   #. Renaming directories is not allowed. 
+   #. It keeps an in-memory cache of the directory structure, so it may not be usable for large file systems. In addition, files added by other applications will not show up until the cache expires.
+   #. The maximum number of listings is 10,000 items.
+   
+   Please keep these limitations in mind when evaluating ``cc-cloudfuse``.
+   
+   You may experience the persistence issues when using ``cc-cloudfuse``, especially when writing large files or writing many files at the same time.
```

Comment:
```suggestion
   You may experience persistence issues when using ``cc-cloudfuse``, especially when writing large files or writing many files at the same time. Unmounting and re-mounting usually resolves this.
```

Maybe this could even be a `note`? So it has the callout styling.